core/app/models/workarea/pricing/discount/conditions/order_total.rb

```diff
@@ -25,7 +25,7 @@ module OrderTotal
           #
           # @return [Boolean]
           #
-          def order_total?
+          def use_order_total?
```

Comment:
I assume this is to not step on the mongoid created method, yes?

Reply:
Yeah, exactly